Description: Python versions prior to 2.6 does not have
 socket.create_connection, and getattr raises an exception for unknown attrs.
Author: Ã–rjan Persson <orange@fobie.net>
Last-Update: 2010-04-05
Index: python-gevent/doc/conf.py
===================================================================
--- python-gevent.orig/doc/conf.py	2010-04-05 16:22:15.260201430 +0200
+++ python-gevent/doc/conf.py	2010-04-05 16:22:51.958610929 +0200
@@ -220,7 +220,7 @@
 del gevent.Greenlet.throw
 del gevent.socket.socket.fd
 for item in gevent.socket.__all__[:]:
-    if getattr(gevent.socket, item) is getattr(socket, item):
+    if hasattr(socket, item) and getattr(gevent.socket, item) is getattr(socket, item):
         gevent.socket.__all__.remove(item)
 
 
